crucible-llvm-syntax/README.md

```diff
@@ -34,4 +34,10 @@ If the numeral `w` is the width of a pointer and `n` is an arbitrary numeral,
 - `load : Alignment -> LLVMType -> Ptr w -> T`: load a value from memory, where the type `T` is determined by the `LLVMType`
 - `store : Alignment -> LLVMType -> Ptr w -> T -> Unit`: store a value to memory, where the type `T` is determined by the `LLVMType`
 
+## Further extensions
+
+The parser hooks can be further customized by passing yet another `ParserHooks`
```

Comment:
Perhaps say:

```suggestion
The LLVM parser hooks can be further customized by passing yet another `ParserHooks`
```

To better distinguish the two types of parser hooks that this sentence refers to.